pages/governmentresumeguide.md

```diff
@@ -0,0 +1,75 @@
+#Government-style Resume Guide
+
+**Full legal name (First Middle Last)** <br>
```

Comment:
Let's change this to `Full name`; if someone goes by a name other than their legal name, they can use the name they go by and just supply the necessary information when it comes time for their background check. Otherwise we set ourselves up to call applicants by the wrong name throughout the hiring process.